.template-lintrc.js

```diff
@@ -29,6 +29,12 @@ module.exports = {
   },
 
   overrides: [
+    {
+      files: ['**/tests/**/*.{js,ts}'],
+      rules: {
+        'eol-last': 'off',
```

Comment:
Can you just clarify why you need to do this?

Reply:
See https://github.com/DazzlingFugu/ember-slugify/actions/runs/3551138491/jobs/5965100031 .

The linter was complaining:

```js
  test('it removes diacritics', async function (assert) {
    await render(hbs`{{remove-diacritics undefined}}`)
    assert.dom().hasText('')
    await render(hbs`{{remove-diacritics ""}}`)
    assert.dom().hasText('')

    // ...
  })
```

and was expecting things to be like this

```js
  test('it removes diacritics', async function (assert) {
    await render(hbs`{{remove-diacritics undefined}}
`)
    assert.dom().hasText('')
    await render(hbs`{{remove-diacritics ""}}
`)
    assert.dom().hasText('')

    // ...
```

which is obviously a big no-no.

Reply:
So weird...